css/properties/counter-reset.json

```diff
@@ -68,7 +70,9 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "≤13.1",
+                "partial_implementation": true,
+                "notes": "The <code>list-item</code> counter is <em>not</em> used correctly when generating content, as it is <em>not</em> fully implemented; see <a href='https://webkit.org/b/260436'>bug 260436</a>."
```

Comment:
What you're saying here is that the `list-item` variable has the wrong value, right?

<img width="323" alt="image" src="https://github.com/user-attachments/assets/2077d3ec-353d-45d4-a4e5-ac1c66967a28">


For reference, this is how it looks in Firefox:

<img width="323" alt="image" src="https://github.com/user-attachments/assets/72a6a9a1-b035-44e0-8188-541f7baf83be">

Reply:
dito 👍